### ISSUE TITLE
Crash on Android at application startup

### ISSUE THREAD

**Issue by zabojad** [jeduan/cordova-plugin-facebook4#712]
**Bug or feature request**

[x] I'm reporting a reproducible issue with the code
[ ] I'm reporting a feature request

**Describe the Bug of feature request**

My app crashes at startup on Android with the following log in logcat:
```
10-10 12:04:19.736 16109-16109/? E/AndroidRuntime: FATAL EXCEPTION: main
    Process: my.app, PID: 16109
    java.lang.NoSuchMethodError: No virtual method getCodelessSetupEnabled()Z in class Lcom/facebook/internal/FetchedAppSettings; or its super classes (declaration of 'com.facebook.internal.FetchedAppSettings' appears in /data/app/my.app-1/base.apk:classes16.dex)
        at com.facebook.marketing.CodelessActivityLifecycleTracker$1.onActivityResumed(CodelessActivityLifecycleTracker.java:83)
        at android.app.Application.dispatchActivityResumed(Application.java:238)
        at android.app.Activity.onResume(Activity.java:1372)
        at org.apache.cordova.CordovaActivity.onResume(CordovaActivity.java:265)
        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1286)
        at android.app.Activity.performResume(Activity.java:6987)
        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4144)
        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4245)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3360)
        at android.app.ActivityThread.access$1100(ActivityThread.java:221)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1794)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:158)
        at android.app.ActivityThread.main(ActivityThread.java:7224)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1230)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1120)
```

**Expected Behavior**

Obviously, not crashing...

**Plugin version, OS, devices, etc**

```
$ cordova platform list
Installed platforms:
  android 6.4.0
  ios 4.5.4
$ cordova -v
8.0.0
$ cordova plugin list
cordova-plugin-badge 0.8.7 "Badge"
cordova-plugin-camera 4.0.2 "Camera"
cordova-plugin-device 2.0.1 "Device"
cordova-plugin-dialogs 2.0.2-dev "Notification"
cordova-plugin-facebook4 1.9.1 "Facebook Connect"
es6-promise-plugin 4.2.2 "Promise"
```

**Comment by zabojad** [comment]
happening only on Android 6.0.1, not on Android >= 7

**Comment by Slavezax** [comment]
We are experiencing same on some phones after upgrading the plugin to latest release. I made some experiments and figured out the problem is the new SDK. This Stack Overflow answers solve it;

https://stackoverflow.com/questions/52113549/app-getting-stuck-with-e-com-facebook-internal-attributionidentifiers

**Comment by peterpeterparker** [comment]
So there is a version v4.36.1 of both Android SDK and they even published a version v4.37.0

I'll update and publish a new version of the plugin referencing these versions with https://github.com/jeduan/cordova-plugin-facebook4/issues/713

I close this issue because it isn't related to the plugin, nothing we could do (= code) on the plugin side

**Comment by peterpeterparker** [comment]
@Slavezax v3.1.0 of the plugin including per default on Android the Facebook SDK v4.37.0 has been published. If you would updated and when you would do so, could you ping me and inform me if you would still face or not this issue? Just for the record, to let me know what's the status. Thx in advance
